docs/manual/src/udl/ext_types_wrapped.md

```diff
@@ -125,15 +124,40 @@ The behavior of the generated scaffolding will be:
 * Calling `take_handle_2` with a value of `-1` will always panic.
 * All other values will return `Ok(ExampleHandle)`
 
-## Custom Foreign Wrappers
+## Custom types in the bindings code
+
+*Note: The facility described in this document is not yet available for the Ruby bindings.*
+
+By default, the foreign bindings just see the builtin type - eg, the bindings will get an integer
+for the `Handle`.
+
+However, custom types can also be converted on the bindings side.  If you
+have a custom Kotlin Handle class you can configure UniFFI to use that for the
+`Handle` type by adding code like this to `uniffi.toml`:
+
+```toml
+[bindings.kotlin.custom_types.Handle]
+# Kotlin custom type name
+type_name = "Handle"
+# Expression to convert the builtin type to a Handle.  In this example, `{}` will be replaced with the int value.
+into_custom = "Handle({})
+# Expression to convert a Handle to the builtin type.  In this example, `{}` will be replaced with the Handle value.
+from_custom = "{}.toString()"
+# Modules we need to import
+imports = [
+  "some.package.Handle",
+]
+```
 
-In the example above, the foreign bindings just see the "wrapped" value - eg, the bindings will
-get an integer for the `Handle`.
+Here's how the configuration works in `uniffi.toml`.
 
-We intend on making it possible to customize this - for example, `serde_json::Value` could be
-transformed into a suitable JSON object in the bindings.
+* Create a `[bindings.{language}.custom_types.{CustomTypeName}]` table to enable a custom type on a bindings side.  This has several subkeys:
```

Comment:
In the context of "any type can be customized", I'm not sure I understand what the point of the`[Custom]` attribute is to the foreign language bindings.

This isn't something for this PR just asking the question early.